assets/src/application/services/theme/theme.ts

```diff
@@ -2,8 +2,22 @@ import { theme as chakraTheme, DefaultTheme } from '@chakra-ui/core';
 
 type CustomTheme = DefaultTheme;
 
+const breakpoints = ['30em', '48.875em', '62em', '80em'];
+
+// aliases
+
+// @ts-ignore
+breakpoints.sm = breakpoints[0];
+// @ts-ignore
+breakpoints.md = breakpoints[1];
+// @ts-ignore
+breakpoints.lg = breakpoints[2];
+// @ts-ignore
+breakpoints.xl = breakpoints[3];
```

Comment:
followed the same approach as chakra does:
https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/theme/theme.js#L21

Reply:
although it works, I did not found smth related to these aliases

Reply:
tried this before:
```
import { theme as chakraTheme, DefaultTheme } from '@chakra-ui/core';

type CustomTheme = DefaultTheme;

const breakpoints = ['30em', '48.875em', '62em', '80em'];

const theme: CustomTheme = {
	...chakraTheme,
	breakpoints,
};

export default theme;
```

Reply:
why not:

```
const breakpoints = {
	sm: '30em',
	md: '48.875em',
	lg: '62em',
	xl: '80em'
};

const theme: CustomTheme = {
	...chakraTheme,
	...breakpoints,
};
```

Reply:
It is crashing the app:

![image](https://user-images.githubusercontent.com/1477453/82260637-4ca7ca00-9966-11ea-9973-2ec12c0be705.png)

Reply:
LOLZ... well alrighty